README.md

```diff
@@ -135,6 +135,7 @@ models:
     model:
       - github.com/99designs/gqlgen/graphql.IntID # a go integer
       - github.com/99designs/gqlgen/graphql.ID # or a go string
+			- github.com/99designs/gqlgen/graphql.UintID # or a go uint
```

Comment:
Is this a tab character or spaces? it should match the lines above.

Reply:
thanks!

Reply:
It seems to have been replaced by IDE. I've fixed it.